src/server/handlers.ts

```diff
@@ -271,7 +271,7 @@ export const SendMessageHandler = (socket: Socket) => {
       /**
        * Cache the sent message in memory a nd persist to db
        */
-      const sentMessage = await addMessage(chatId, {
+      const sentMessage = await addMessage(room, {
```

Comment:
leave it as chatId

Reply:
This is a naming error. The client is sending the room, and when I log the chatId, it's showing undefined. we can use chatId: room, but I found it more suitable to simply use room. If you prefer to keep chatId, I'll change it.

Reply:
use chatId: room

Reply:
got it